io.spectra.sudoku_solver.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id" : "io.tachyon.sudoku_solver",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "sudoku-solver",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+              {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
```

Comment:
There should be a cleanup rule to remove it all afterwards since it's just a build dependency.

Reply:
Sorry for that, https://jwestman.pages.gitlab.gnome.org/blueprint-compiler/flatpak.html didn't mention anything about cleanup so i didn't know about it.

Reply:
I don't know how to setup a cleanup rule for blueprint compiler, can you point me to some documentation about it.

Reply:
Taking reference from https://github.com/Foldex/AdwSteamGtk/blob/main/io.github.Foldex.AdwSteamGtk.json#L43
I've added cleanup rule for blueprint-compiler in https://github.com/cyberphantom52/flathub/commit/00d0a2be46edafcee90f2509c2b4e175be9a0b42

Reply:
I'm going to keep this conversation unresolved as i'm not sure if this is the correct way.

Reply:
Resolving

Ref: https://github.com/flathub/flathub/pull/3805#discussion_r1070665768